infrastructure/w3id/src/logs/log.types.ts

```diff
@@ -0,0 +1,35 @@
+export type LogEvent = {
+	id: string;
+	versionId: string;
+	versionTime: Date;
+	updateKeys: string[];
+	nextKeyHashes: string[];
+	method: `w3id:v${string}`;
+	proof?: string;
+};
+
+export enum LogEvents {
+	Rotation = 0,
+	Genesis = 1,
+}
```

Comment:
`LogEvents` and `LogEvent` are very ambiguous, shall rename to EventType instead